client/state/account-recovery/test/test-data.js

```diff
@@ -0,0 +1,13 @@
+const dummyData = {
+	email: 'dummytest@a8ctest.com',
```

Comment:
I didn't know there was this handy domain :)
Fixed by ec0777fae31c1c43729f2596316a1e1ec372815c.